internet-service-provider/src/main/java/org/epam/nahorniak/spring/internetserviceprovider/controller/ServiceController.java

```diff
@@ -0,0 +1,55 @@
+package org.epam.nahorniak.spring.internetserviceprovider.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.epam.nahorniak.spring.internetserviceprovider.controller.dto.ServiceDto;
+import org.epam.nahorniak.spring.internetserviceprovider.service.ServicesService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
+public class ServiceController {
+
+    private final ServicesService servicesService;
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping(value = "/service")
```

Comment:
I would use @RequestMapping("/service") instead of duplicate this url in each method

Reply:
Fixed , ty)